nuxt.config.js

```diff
@@ -83,7 +83,11 @@ export default {
   axios: {},
 
   // Build Configuration (https://go.nuxtjs.dev/config-build)
-  build: {},
+  build: {
+    options: {
+      fix: true,
```

Comment:
I couldn't find documentation on this. Can you provide a link?

Reply:
https://stackoverflow.com/a/54210573

Reply:
I don’t think the way you configured it here actually does anything. 

You included the `options: { fix: true }` bit, but not as part of a modification for the eslint loader config. Let’s remove this for now, and if you need to include this for your set up, we can do it as a separate PR.